src/drive/web/modules/viewer/Panel/getPanelBlocks.spec.jsx

```diff
@@ -0,0 +1,59 @@
+import getPanelBlocks from './getPanelBlocks'
+
+const block1Component = jest.fn()
+const block2Component = jest.fn()
+
+describe('getPanelBlocks', () => {
+  it('should return only blocks with truthy condition', () => {
+    // with two truthy component
+    expect(
+      getPanelBlocks({
+        panelBlocksSpecs: {
+          block1: {
+            condition: () => true,
+            component: block1Component
+          },
+          block2: {
+            condition: () => true,
+            component: block2Component
+          }
+        }
+      })
+    ).toMatchObject([block1Component, block2Component])
+
+    // with one truthy component
+    expect(
+      getPanelBlocks({
+        panelBlocksSpecs: {
+          block1: {
+            condition: () => false,
+            component: block1Component
+          },
+          block2: {
+            condition: () => true,
+            component: block2Component
+          }
+        }
+      })
+    ).toMatchObject([block2Component])
+
+    // with no truthy component
+    expect(
+      getPanelBlocks({
+        panelBlocksSpecs: {
+          block1: {
+            condition: () => false,
+            component: block1Component
+          },
+          block2: {
+            condition: () => false,
+            component: block2Component
+          }
+        }
+      })
+    ).toMatchObject([])
```

Comment:
Nit: I am not sure that 3 separates expect are necessary. I think all the behavior could be tested with only one mixing condition => true/false.

Reply:
@ptbrowne c'est à dire ? Désolé je ne vois pas trop comment faire un `setup`simple qui permet de gérer tous les cas

Reply:
Pas de soucis. Sans `setup`, je pensais que le 

```
getPanelBlocks([
  { condition: () => true },
  { condition: () => false },
  { condition: () => false },
  { condition: () => true },
])
```

suffirait à tester tout les cas. C'est un nit ;)